schema/sale/sorts.js

```diff
@@ -40,6 +40,12 @@ export default {
       ELIGIBLE_SALE_ARTWORKS_COUNT_DESC: {
         value: '-eligible_sale_artworks_count',
       },
+      TIMELY_AT_NAME_ASC: {
```

Comment:
Yep, this is correct. This is mostly useful when constructing queries in GraphiQL, `TIMELY_AT_NAME_ASC` will be presented as an autocomplete option.